packages/server/src/router.ts

```diff
@@ -1,13 +1,23 @@
 import * as Router from "koa-router";
 
 import { getBooking, getBookings } from "./controllers/bookings";
+import { createAccount, getAccount } from "./controllers/accounts";
+import { validateCreateAccountBody } from "./middlewares/validate";
+import { login } from "./controllers/auth";
+import { authenticate } from "./middlewares/authenticate";
 
 export const router = new Router();
 
 router.get("/healthz", (ctx) => {
   ctx.body = { ok: true };
 });
 
-router.get("/bookings", getBookings);
+router.get("/bookings", authenticate, getBookings);
 
-router.get("/bookings/:bookingId", getBooking);
\ No newline at end of file
+router.get("/bookings/:bookingId", authenticate, getBooking);
+
+router.post("/accounts", validateCreateAccountBody, createAccount);
+
+router.get("/accounts/:accountId", getAccount);
```

Comment:
Shouldn't this route be protected ? 
```suggestion
router.get("/accounts/:accountId", authenticate , getAccount);
```

Reply:
Oui il faut qu'on se fasse un récap à l'occas des endpoints qu'il nous reste à faire et la partie administrateur etc